api-channel-jira/src/main/java/com/synopsys/integration/alert/api/channel/jira/distribution/custom/MessageReplacementValues.java

```diff
@@ -25,17 +21,24 @@
     private final String componentVersionName;
     private final String severity;
 
-    public static JiraCustomFieldReplacementValues trivial(LinkableItem provider) {
-        return new JiraCustomFieldReplacementValues(provider.getLabel(), DEFAULT_REPLACEMENT, null, null, null, null);
+    public static MessageReplacementValues trivial(String providerLabel, String projectName) {
+        return new MessageReplacementValues(
+            providerLabel,
+            projectName,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
+            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE
+        );
     }
 
-    public JiraCustomFieldReplacementValues(
+    public MessageReplacementValues(
         String providerName,
         String projectName,
-        @Nullable String projectVersionName,
-        @Nullable String componentName,
-        @Nullable String componentVersionName,
-        @Nullable String severity
+        String projectVersionName,
+        String componentName,
+        String componentVersionName,
+        String severity
```

Comment:
These values a can still be null if they are not provided. Do we no longer need the @Nullable annotation on them?

Reply:
Example in JiraIssueCreator::113, we could still be passing in null values since we don't use the "trivial" static method